component/web/security/src/main/java/org/exoplatform/web/login/LoginHandler.java

```diff
@@ -254,6 +254,10 @@ private String getUserNameByEmail(String email,
         if (users != null && users.getSize() > 0) {
           return users.load(0, 1)[0].getUserName();
         }
+        else {
+          LOG.warn("Can not get users by email");
```

Comment:
It's better to avoid adding such information as Warning in log since it's a normal behavior and not an unexpected behavior from system. (you can add a `log.debug` if useful for troubleshooting)